### ISSUE TITLE
About connects CRM

### ISSUE THREAD

**Issue by huang1937** [jlattimer/D365DeveloperExtensions#27]
Hello Jlattimer, 
I wrote a plug-in and released it with D365Developer. I click on the solution to select the release plug-in, enter the information that connects CRM, click on the connection, after 1 minutes, vs2015 automatically crashes and restarts vs. My CRM server is local, and the SDK version is 8.1.*

**Comment by huang1937** [comment]
yes,using latest version,no special about plug-in. use PluginBase. I changed a new computer to connect to CRM tonight, but when vs2017 was released, it collapsed or restarted. In this plug-in solution, I have a business logic class library. The plug-in class library refers to the business logic class library. I used the ILMerge button to merge before publishing. I used the last version of v1.3.** to release plug-ins.

**Comment by jlattimer** [comment]
If it possible to get a solution that reproduces the issue posted somewhere I can download and investigate?
